{
  "uid" : "98e9ccb35becdb9b",
  "name" : "test_broken",
  "fullName" : "test_alluredemo#test_broken",
  "historyId" : "fb799eecaee64eb20ac70a0047ddd7ff",
  "time" : {
    "start" : 1616899047699,
    "stop" : 1616899047699,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "Exception: oops",
  "statusTrace" : "def test_broken():\n    \n>       raise Exception('oops')\nE       Exception: oops\n\ntest_alluredemo.py:21: Exception",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "test_alluredemo"
  }, {
    "name" : "host",
    "value" : "SKY-20200805OYN"
  }, {
    "name" : "thread",
    "value" : "2164-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_alluredemo"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "98e9ccb35becdb9b.json",
  "parameterValues" : [ ]
}